h/templates/activity/search.html.jinja2

```diff
@@ -105,6 +105,191 @@
 </div>
 {% endmacro %}
 
+{% macro tags_section() %}
+<section class="search-result-sidebar__section">
+  <h3 class="search-result-sidebar__subtitle">
+    {% trans %}Tags{% endtrans %}
+    <span class="search-result-sidebar__subtitle-count">
+      {{ aggregations['tags'] | length }}
+    </span>
+  </h3>
+  <div class="search-result-tags">
+    {% for tag in aggregations.tags %}
+      <div class="search-result-tag js-tag">
```

Comment:
It doesn't look like there are any controllers registered for the `.js-tag` selector?